lib/modules/dosomething/dosomething_api/resources/signup_resource.inc

```diff
@@ -189,6 +189,25 @@ function _signup_resource_index($user, $users, $campaigns, $count, $page, $compe
     'runs' => $runs,
   ];
 
+  if (variable_get('rogue_collection', FALSE)) {
+    if ($user) {
+      $northstar_user_id = dosomething_user_get_northstar_id($user->uid);
+    }
+
+     $rogue = dosomething_rogue_client();
+
+     $response = $rogue->getActivity([
+       'filter' => [
+         'northstar_id' => $northstar_user_id,
+         'campaign_id' => $campaigns[0],
+       ],
+     ]);
+
+     if ($response) {
+      return $response;
```

Comment:
@DFurnes this doesn't return exactly what Phoenix's `GET /signups` returns but not sure what's the best wayto do this. 

Should I build out a new endpoint specifically for this in Rogue that mirrors the Phoenix response? I feel like that is the best answer but also might be a little overkill with Rogue's `/activity` endpoint (especially in this scenario where we just want to see if a Signup exists)?